releasenotes/notes/whitelist-tracer-metrics-82a29f90053042b8.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Add datadog.tracer to the list of standard statsd prefixes. Tracer metrics are published with this prefix.
```

I'd mention how adding this to the list of standard prefixes impacts the user, maybe something like:

```suggestion
    Do not apply the metric namespace configured under `statsd_metric_namespace` to dogstatsd metrics prefixed with `datadog.tracer`. Tracer metrics are published with this prefix.
```